examples/workspace/driver_navigator/main.py

```
@@ -0,0 +1,125 @@
+print(f"Created node: navigator (WezTerm) in {cwd}")
+
+# =============================================================================
+# Suggestions Node - AI-powered command suggestions
+# =============================================================================
+
+SuggestionNode(
+    id="suggestions",
+    session=session,  # noqa: F821
+    api_key=os.environ.get("OPENROUTER_API_KEY", ""),
+    model="google/gemini-3-flash-preview",
+    debug_dir="/tmp/nerve-debug",
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/nerve-debug"

(S108)